### DISCUSSION TITLE
The issue of IE compatibility requires everyone's help

### DISCUSSION THREAD

**Discussion by JACK-YUNS** [webpack/webpack#16844] [Q&A]
"I have a Vue2 project that uses module federation. It needs to be compatible with IE, and it can run normally in Google Explorer. However, in IE, when it loads js modules shared by other applications, the IP address it loads is incorrect. The port of the main application is 9700, and the port of other applications is 9702.",
![image](https://user-images.githubusercontent.com/28399612/226083255-f0a43f84-021d-49b2-a0d5-75c859e7ed60.png)
![image](https://user-images.githubusercontent.com/28399612/226083307-bad99feb-e5a1-4492-b357-40f892468415.png)

"I don't understand why the port he loaded is not 9700, but 9702."

**Comment by JACK-YUNS** [comment]
Now the request port for this 404 is 9700. I hope it is 9702

**Comment by alexander-akait** [comment]
Which IE support do you need?

**Reply by JACK-YUNS** [reply]
ie11

**Reply by JACK-YUNS** [reply]
I suspect that when webpack references js components in a project, it cannot accurately determine whether it is a js provided by a remote container, resulting in sometimes being able to load and sometimes not being able to load

**Reply by alexander-akait** [reply]
Please provide error message from console, thank you
